testcases/open_posix_testsuite/conformance/interfaces/munmap/2-1.c

```diff
@@ -4,12 +4,15 @@
  * of this license, see the COPYING file at the top level of this
  * source tree.
  *
- * If there are no
- * mappings in the specified address range, then munmap() has no effect.
+ * If there are no mappings in the specified address range, then munmap() has
+ * no effect. To get a valid address range which is safe to call munmap() on
+ * we first map some arbitrary memory allowing the OS to select the address
+ * then unmap it. We then call munmap() on the same address again to perform
+ * the test.
  *
  */
 
-#define _XOPEN_SOURCE 600
+#define _DEFAULT_SOURCE
 
```

Comment:
I started another version which doesn't use `MAP_ANONYMOUS`, but it won't link without adding some libs to munmaps Makefile https://github.com/linux-test-project/ltp/compare/master...richiejp:munmap_shm_open?expand=1